### PR TITLE
keep clean main.go, multiplexing init func

### PR DESCRIPTION
**Pull request by liyongjing**
if have many api, maybe multiplexing init func

### CODE REVIEW

**Comment by ardan-bkennedy** [comment]
Don’t want to hide this outside of main. This is very important to have as transparent and readable. 

I want to avoid encapsulation anything from main.

**Comment by liyongjing** [comment]
oh, close it

File: business/config/entry.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package config
+
+import "go.uber.org/zap"
+
+func RunApi(log *zap.SugaredLogger, build string) error {
```
Comment (andreygrehov) [inline business/config/entry.go:5]: nit: should be RunAPI.

**Comment by ardan-bkennedy** [comment]
Good point. I’ll make that change.

**Pull request closed without merging**